cmd/control_loop/manifest_reader/manifest_reader.go

```diff
@@ -0,0 +1,43 @@
+package manifest_reader
+
+import (
+	"io/ioutil"
+	"fmt"
+	"gopkg.in/yaml.v2"
+)
+
+type Dependency struct {
```

Comment:
These direct definitions work for our demo and exploration, but to prepare to store the manifest as an artifact in the Registry, I think we should eventually replace these Go structs with a Protocol Buffer definition - we can still read that from YAML files but we would store it in the Registry as a serialized protobuf.

Reply:
Created a follow-up issue to track this.